### ISSUE TITLE
A problem about time out 

### ISSUE THREAD

**Issue by whs1995** [bitcoin-sv/bitcoin-sv#156]
## these error is from bitcoind.log 
`2020-02-19 13:33:29 [bitcoin-msghand] New outbound peer connected: version: 70015, blocks=622959, peer=45
2020-02-19 13:34:30 [bitcoin-msghand] New outbound peer connected: version: 70015, blocks=622959, peer=46
2020-02-19 13:35:31 [bitcoin-msghand] New outbound peer connected: version: 70015, blocks=622959, peer=47
2020-02-19 13:36:31 [bitcoin-msghand] New outbound peer connected: version: 70015, blocks=622959, peer=48
2020-02-19 13:37:32 [bitcoin-msghand] New outbound peer connected: version: 70015, blocks=622959, peer=49
2020-02-19 13:37:56 [bitcoin-opencon] connect() to [2001:250:fe01:130:148b:c481:c8f2:df86]:8333 failed: Network is unreachable (101)
2020-02-19 13:38:03 [bitcoin-opencon] connect() to 185.216.35.68:8333 failed after select(): Connection refused (111)
2020-02-19 13:38:33 [bitcoin-msghand] New outbound peer connected: version: 70015, blocks=622959, peer=50
2020-02-19 13:39:33 [bitcoin-msghand] New outbound peer connected: version: 70015, blocks=622960, peer=52
2020-02-19 13:41:35 [bitcoin-net] version handshake timeout from 53
`
## this error is found when I try to request RPC
`timeout reading data from server`

**Comment by bitcartel** [comment]
Looks like the node was connecting to BCH peers as BSV Block 622959 was only mined on Feb 21, while BCH Block 622959 was mined on Feb 19 per the log output above.

**Comment by Danconnolly** [comment]
Yes. Try `bitcoin-cli addnode 68.183.207.240 add` to ensure that it connects to a BSV node.

**Comment by whs1995** [comment]
Thanks to reply, so far, he seems to play a role, now running stability

**Comment by whs1995** [comment]
> Looks like the node was connecting to BCH peers as BSV Block 622959 was only mined on Feb 21, while BCH Block 622959 was mined on Feb 19 per the log output above.
Yes, you are right, thanks for your reply!

**Comment by scrawld** [comment]
@Danconnolly Are there more stable nodes
